### DISCUSSION TITLE
Instant Preview not working

### DISCUSSION THREAD

**Discussion by duck-irl** [codesandbox/codesandbox-client#6492] [Q&A]
Hello! Instant preview is not working for me even though I have the setting turned on.
I'm using a static site, and it only refreshes when I save the file or click the refresh button.

<img width="681" alt="Screen Shot 2022-02-26 at 6 19 17 PM" src="https://user-images.githubusercontent.com/94648379/155862014-063b47e3-a995-46a3-9c24-4cd5737f88ff.png">

**Comment by JamesACS** [comment]
Hey @duck-irl - I'm sorry to hear about this. 

Are you blocking third party cookies, or do you have an ad blocker enabled? Both of these can cause issues with CodeSandbox
